nncf/quantization/fake_quantize.py

```diff
@@ -328,7 +323,7 @@ def _calculate_scaled_parameters(
     num_bits = quantizer_config.num_bits
     level_low, level_high = calculate_symmetric_level_ranges(num_bits - 1, signed=True, narrow_range=False)
     levels = get_num_levels(level_low, level_high)
-    input_low, input_high = symmetric_range(min_values, max_values, levels, quantizer_config, quant_group)
+    input_low, input_high = symmetric_range(min_values, max_values, levels, quantizer_config, narrow_range)
```

Comment:
@kshpv, please confirm `half_range` calculations are correct. input_low = -input_hight before despite the value of the narrow_range before (due to quant_group == WEIGHTS). And now, for a synthetic case, impossible with `cpu.json` config (symmetric weights + narrow_range==False), calculations were altered